SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs

```diff
@@ -95,19 +100,26 @@ protected override string GetOsVersion()
 
         protected override string GetAppVersion()
         {
-#if NET461
-            // Get ClickOnce version or fall back to assembly file version. ClickOnce does not exist on .NET Core.
-            if (System.Deployment.Application.ApplicationDeployment.IsNetworkDeployed)
-            {
-                return System.Deployment.Application.ApplicationDeployment.CurrentDeployment.CurrentVersion.ToString();
-            }
-#endif
+            /*
+             * If the AssemblyInformationalVersion is not applied to an assembly,
+             * the version number specified by the AssemblyFileVersion attribute is used instead.
+             */
             return Application.ProductVersion;
```

Comment:
Why are we removing the ClickOnce support for the main version field?

Reply:
That's why It's marked as "do not merge" - it's under discussion now. Currently, I changed versions in following way:
App Version - human readable
App Build - unique build/deployment

Reply:
This will break #1088 use case as Analytics uses only appVersion. We cannot make such a risky change right now.

I agree appBuild needs to be different than appVersion when ClickOnce is not used (the rest of the patch) but I think for ClickOnce we need to use it for both to avoid breaking scenarios.